### DISCUSSION TITLE
No configure renovate PR created

### DISCUSSION THREAD

**Discussion by YentlFrickx** [renovatebot/renovate#25554] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

I setup the app on github with access to 2 repositories (added a second one to test if there was a problem with the first one). However Renovate is not generating the 'Configure Renovate' Pull-Request in these repositories. My first repository contains helm values and kubernetes charts, so should be valid content for Renovate to work on (private repository though), second one is a go application: https://github.com/YentlFrickx/kubernetes-ddns .

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
Go to https://developer.mend.io to finish the onboarding process

**Reply by YentlFrickx** [reply]
Forgot to mention it but I went there but it says 'There are no GitHub organizations connected to this account', I am trying to connect my personal repositories so not a separate organization, but I didn't see anywhere that that wouldn't be possible.

**Reply by rarkins** [reply]
OK, we'll take a look and see if we can identify the problem. Installations on personal accounts are definitely possible

**Reply by rarkins** [reply, marked as answer]
We've identified a backlog in our installation sync processing and it's currently being fixed - it should be resolved within the hour once the queue drains

**Reply by YentlFrickx** [reply]
Perfect, thanks! Will close this once the PR is there

**Reply by rarkins** [reply]
You will probably need to go to developer.mend.io again now that the account has sync'd and finish the onboarding of your account in order to receive onboarding PRs.
